src/components/Kurlsh.js

```diff
@@ -591,8 +591,19 @@ class Kurlsh extends React.Component {
 
   getLabel = name => ({ version }) => {
     if (version === "latest") {
-      const latest = this.state.versions[name][1];
-      version = `latest (${latest.version})`;
+      if (name === "kubernetes") {
```

Comment:
i'll prefer this to be written as a separate function and called inside this one.

``` 
 generateVersionLabel = (name, version) => {
    if (version === "latest") {
      if (name === "kubernetes") {
        const latest = this.state.versions[name][2]; // for k8s, the first version is a ".x" version
        version = `latest (${latest.version})`;
      } else {
        const latest = this.state.versions[name][1];
        version = `latest (${latest.version})`;
      }
    } else if (version.endsWith(".x")) {
      const versionIndex = this.state.versions[name].findIndex((element) => element.version === version);
      if (this.state.versions[name].length > versionIndex) { // if there is a member of the array after the one specified
        const next = this.state.versions[name][versionIndex+1]
        version = `${version} (${next.version})`
      }
    } 
    return version;
  }
  ```
  
  and getLabel after looks more cleaner:
  ```
    getLabel = name => ({ version }) => {
    const generatedVersion = this.generateVersionLabel(name, version);
    return (
      <div className="versionLabel--wrapper">
        <span className="versionLabel" style={{ fontSize: 14 }}>{generatedVersion}</span>
      </div>
    );
  }
  ```